### PR TITLE
#166789987 Set up API Documentation with Swagger

### PR DESCRIPTION
**Pull request by elemanhillary**
#### What does this PR do?
- Configure swagger documentation.

#### Description of Task to be completed?
- Ensure documentation with swagger is Enabled.

#### How should this be manually tested?
- clone the repo on your computer
- `<app-name>.herokuapp.com/docs`

#### What are the relevant pivotal tracker stories?
[#166789987](https://www.pivotaltracker.com/story/show/166789987)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+import http from 'http';
+import path from 'path';
+import methods from 'methods';
+import express from 'express';
+import bodyParser from 'body-parser';
+import session from 'express-session';
+import cors from 'cors';
+import passport from 'passport';
+import errorhandler from 'errorhandler';
+import mongoose from 'mongoose';
```
[the remark this answers is not in the document]
Comment (Hervera) [reply index.js]: Why have you included mongoose? Are we going to use that in this project?

**Comment by elemanhillary** [comment]
> We will be using Postgres, why have you imported mongoose?

mongoose was already there, don't worry about it. We shall remove it per the instructions or requirements. Thanks for the suggestion.

**Comment by niyoceles** [comment]
> HoundCI is still complaining but it's the issue of how houndCI was set up, it can be merged!!

I am working on it.

**Comment by elemanhillary** [comment]
> > HoundCI is still complaining but it's the issue of how houndCI was set up, it can be merged!!
> 
> I am working on it.

@niyoceles it's okay, you can actually work on other tasks till its merged. Thanks

File: swaggerSetUp/swaggerSetup.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import swaggerDoc from 'swagger-jsdoc';
+
+const swaggerDefinition = {
+  info: {
+    title: 'Authors Haven',
+    version: '1.0.0',
+    description: 'Create a community of like minded authors to foster inspiration and innovation by leveraging the modern web',
+  },
+  host: 'example-authors-haven.herokuapp.com',
```
Comment (elemanhillary) [inline swaggerSetUp/swaggerSetup.js]: change this to `92-explorers-authors-haven`

**Pull request merged**